Documentation/Tips.md

```diff
@@ -5,22 +5,23 @@ Tips
 * When you are using Rx, first try to compose built-in operators.
 * If using some combination of operators often, create your convenience operators.
 
-e.g.
-```swift
-extension ObservableType where E: MaybeCool {
-
-    public func coolElements()
-        -> Observable<E> {
-          return filter { e -> Bool in
-              return e.isCool
-          }
+    e.g.
+
+    ```swift
+    extension ObservableType where E: MaybeCool {
+
+        public func coolElements()
+            -> Observable<E> {
+            return filter { e -> Bool in
+                return e.isCool
+            }
+        }
     }
-}
-```
+    ```
```

Comment:
Can we un-indent everything here ?

Reply:
Since this block is belongs to an item in the list, it should be indented.

See the screen shot below:

![Pasted_Image_2022_5_12__13_25](https://user-images.githubusercontent.com/513082/167998328-ca2c96cf-1d69-4c04-9d64-79edd18282bd.png)

Reply:
Makes sense, thanks!